### ISSUE TITLE
【deepin_V20.8】【一般】【有空】【github反馈】【DDE】【控制中心】[Repository/Mirror] Issues when changing the mirror site

### ISSUE THREAD

**Issue by xoascf** [linuxdeepin/developer-center#3783]
### SUMMARY | 问题概要

When I turn off the "Smart Mirror Switch" option I get the list of mirrors to select in the Control Center, however, I always see "官方CDN（加速）源" mirror selected when it is not.

https://user-images.githubusercontent.com/28744392/220222472-20ed61b5-6660-4d3b-bfa3-bd8cec8b9aaa.mp4

### SOFTWARE/OS VERSIONS | 软件/系统版本信息

Distribution: Deepin 20.8
Package: dde-control-center 5.5.156-1+rb1

### STEPS TO REPRODUCE | 复现步骤

1. Open Control Center.
2. Turn off Smart Mirror Switch in Update Settings.
3. Select any mirror site in the Mirror List.

**Comment by raspin0** [comment]
Duplicate https://github.com/linuxdeepin/developer-center/issues/2755

**Comment by xoascf** [comment]
> Duplicate #2755

We know, it was created so @justforlxz could fix it, and he could track the issue since he said he could fix it.

**Comment by BLumia** [comment]
Could you check if it also happens on v23 alpha2? If not, we can close this one and track the issue in #2755 instead.

**Comment by SuperEffie** [comment]
> Could you check if it also happens on v23 alpha2? If not, we can close this one and track the issue in #2755 instead.

This function is blocked on v23 alpha2,So we can track issue  #2755 instead.

**Comment by SuperEffie** [comment]
> ### SUMMARY | 问题概要
> When I turn off the "Smart Mirror Switch" option I get the list of mirrors to select in the Control Center, however, I always see "官方CDN（加速）源" mirror selected when it is not.
> 
> ### SOFTWARE/OS VERSIONS | 软件/系统版本信息
> Distribution: Deepin 20.8 Package: dde-control-center 5.5.156-1+rb1
> 
> ### PRE-REQUIREMENT | 预置条件
> _No response_
> 
> ### STEPS TO REPRODUCE | 复现步骤
> 1. Open Control Center.
> 2. Turn off Smart Mirror Switch in Update Settings.
> 3. Select any mirror site in the Mirror List.
> 
> ### OBSERVED RESULT | 观察到的结果
> _No response_
> 
> ### EXPECTED RESULT | 期望的结果
> _No response_
> 
> ### ADDITIONAL INFORMATION | 额外补充
> _No response_

Thanks for your feedback,We have tracked this problem in another issue. You can pay attention to it,the link is #2755 and #3626. Now I will close your issue first. 
Welcome to feedback more issues for us~
